README.md

```diff
@@ -225,3 +237,21 @@ Open the template from `project` folder with Android Studio. Now you can directl
 Note 1: If you make changes to the code of the project make sure the project still compiles with `buildAllRelease`. 
 
 Note 2: If you make changes to the project creation script them make sure that the script still runs.
+
+#### CI
```

Comment:
Do we need this here? I'd suggest to just add comments into the codemagic.yaml file. This section will be out of date as soon as someone updates one of the workflows. If the comments are in the codemagic.yaml itself, then there is a chance that someone will update them